src/main/java/io/appium/java_client/service/local/flags/GeneralServerFlag.java

```diff
@@ -119,7 +119,13 @@
     /**
      * Enables NodeJS memory dumps collection feature.
      */
-    ENABLE_HEAP_DUMP("--enable-heapdump");
+    ENABLE_HEAP_DUMP("--enable-heapdump"),
+    /**
+     * Allow a list of features which are considered insecure and must be turned on explicitly by system administrators.
+     * Default: []
+     * Sample: --allow-insecure=foo,bar
+     */
+    ALLOW_INSECURE("--allow-insecure");
```

Comment:
do we also have a constant for `--deny-insecure`?

Reply:
No. I added it now and changed this PR description.